bundle/Resources/config/kernel.yml

```diff
@@ -2,3 +2,9 @@ system:
     default:
         fielddefinition_edit_templates:
             - { template: "EzSystemsRepositoryFormsBundle:ContentType:field_types.html.twig", priority: 0 }
+        location_view:
+            content_name:
+                landing_page:
+                    template: "EzSystemsRepositoryFormsBundle:content_name:content_name.html.twig"
+                    match:
+                        Identifier\ContentType: [landing_page,folder]
```

Comment:
Problem: I see no way to match all/any. I don't care which content type the linked location refers to. But the yml demands a match section. Is there a way to use template X when viewType Y is used, no matter what content we're looking at?